flang/lib/Optimizer/Builder/Runtime/Assign.cpp

```diff
@@ -7,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "flang/Optimizer/Builder/Runtime/Assign.h"
-#include "../../../runtime/assign.h"
+#include "flang/../../../flang/runtime/assign.h"
```

Comment:
Doesn't just adding a fourth `../` fix the problem? `runtime` is a sibling of `lib` after all.

Reply:
Yes.  Adding a fourth `../` works too.  I personally find it less confusing to use `flang/`.

Reply:
Actually, I find it more confusing. It's using the `-I` search path to find the `.../flang/include/flang/` path, then backing up to `.../` then adding the "other" `flang` directory (at the top-level). The preexisting notion was to just backtrack from the directory in which this `.cpp` file is located to the right header file.

I better solution overall would be to have the runtime header files in the include directory like the other exported headers, of course.